src/cluster/index.js

```diff
@@ -159,6 +159,24 @@ module.exports = class Cluster {
     }
   }
 
+  /**
+   * @public
+   * @param {string[]} topics
+   * @return {Promise}
+   */
+  async addMultipleTargetTopics(topics) {
```

Comment:
Doesn't matter much, but why not just name this `addTargetTopics`?

Reply:
I think the version with the plural is hard to spot and prone to typos, I was experimenting with different alternatives before I settled for this name.